cmake/define-ia2-wrapper.cmake

```diff
@@ -1,13 +1,36 @@
+# We need to get the system headers specifically from clang
```

Comment:
Why does this need to be specifically from clang?

Reply:
This is actually from the video player PR (#246), so should be addressed there. This change is built on top of that.

I think the reason is that we are parsing with clang, and on some systems clang can't parse the gcc compiler-rt headers? @ahomescu might know for sure.